pytest_rerunfailures.py

```diff
@@ -106,6 +115,86 @@ class RerunPlugin(object):
 
     def __init__(self):
         self.tests_to_rerun = set([])
+        self.testrun_stats = {}
+        self.rerun_stats = {
+            'rerun_tests': [],
+            'total_failed': 0,
+            'total_reruns': 0,
+            'total_resolved_by_reruns': 0
+        }
+
+    def _failed_test(self, nodeid):
+        """
+        Template format for failed test stat
+
+        Returns
+        -------
+        dict
+        """
+        stat = self.testrun_stats.get(nodeid, {
+            'nodeid': nodeid,
+            'status': 'failed',
```

Comment:
why is it failed by default?

Reply:
By default we interested and track only `failed` tests. After reruns they could be transformed to  `flake` status.